### PR TITLE
Vikas/azure

### PR DESCRIPTION
**Pull request by vikas0212git**
updated to push to blob storage

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -48,5 +92,25 @@ steps:
...
+
+- task: Bash@3
+  displayName: 'Uplaod to Blob'
+  condition: and(succeeded(), eq(variables['NUGET_PUBLISH_FLAG'], 'true'))
+  inputs:
+    targetType: 'inline'
+    script: |
+      export AZCOPY_SPA_CLIENT_SECRET=$(AZURESDKPARTNERDROPS-SERVICE-PRINCIPAL-KEY)
+      azcopy login --service-principal --application-id $(APPID)
+      azcopy copy "$(Build.ArtifactStagingDirectory)/Microsoft.Azure.IoT.Edge.Function.$(PACKAGEVERSION).nupkg"  "https://azuresdkpartnerdrops.blob.core.windows.net/drops/azure-iot-edge-tools/dotnet/$(PACKAGEVERSION)/Microsoft.Azure.IoT.Edge.Function.$(PACKAGEVERSION).nupkg"
```
Comment (mhshami01) [inline azure-pipelines.yml:115]: I think we need one more directory between 'dotnet' and '$(PACKAGEVERSION) - something unique to identify the package.

Comment (vikas0212git) [reply azure-pipelines.yml:115]: will it be able to release if add a extra directory as it was expecting "<team>/dotnet/<version>"

File: azure-pipelines.yml [hunk 2]
```diff
@@ -48,5 +92,25 @@ steps:
     tagSource: 'manual'
     tag: '$(Build.SourceBranch)'
     releaseNotesSource: 'input'
-    isPreRelease: false
```
Comment (marianan) [inline azure-pipelines.yml:51]: >false [](http://example.com/codeflow?start=18&length=5)

should we keep this flag to mark the release as a prerelease?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -48,5 +92,25 @@ steps:
...
+
+- task: Bash@3
+  displayName: 'Uplaod to Blob'
+  condition: and(succeeded(), eq(variables['NUGET_PUBLISH_FLAG'], 'true'))
+  inputs:
+    targetType: 'inline'
+    script: |
+      export AZCOPY_SPA_CLIENT_SECRET=$(AZURESDKPARTNERDROPS-SERVICE-PRINCIPAL-KEY)
+      azcopy login --service-principal --application-id $(APPID)
+      azcopy copy "$(Build.ArtifactStagingDirectory)/Microsoft.Azure.IoT.Edge.Function.$(PACKAGEVERSION).nupkg"  "https://azuresdkpartnerdrops.blob.core.windows.net/drops/azure-iot-edge-tools-templates-module/dotnet/$(PACKAGEVERSION)/Microsoft.Azure.IoT.Edge.Function.$(PACKAGEVERSION).nupkg"
```
Comment (marianan) [inline azure-pipelines.yml]: >module [](http://example.com/codeflow?start=202&length=6)

this should say function as it is publishing the function template i think?

**Review by marianan — APPROVED** [review]
:shipit:

**Pull request merged**